src/Output.f90

```diff
@@ -1002,8 +996,20 @@ subroutine OutputSolutionData_NetCDF(RunParams, filename, grid)
 
          if (tile_i < tile_left) tile_left = tile_i
          if (tile_j < tile_bottom) tile_bottom = tile_j
+
+         call AddToOrderedVector(TileCols, tile_i, repeat=.FALSE.)
```

Comment:
Could this be solved more simply by replacing these calls with `nXtiles = nXtiles + 1` etc?

Reply:
Was expecting this comment to lie above the other one. Still getting used to this!

Reply:
What about tracking the max/min tile_{i,j} and taking their difference? Or is the point that you don't want to store sections between disconnected flow patches?

Reply:
Yes, this would work, and is much easier.

The disconnected patches aren't dealt with in the netcdf.  Perhaps they could/should be?  but maybe not a priority.

I'll change to this.

Reply:
I suppose 'disconnected' netcdfs should never become unmanageably large because if they were, the user ought to have been conducting two separate simulations in the first place.